### ISSUE TITLE
ISBAT Carousel image spacing incorrect, navigation buttons hidden behind images

### ISSUE THREAD

**Issue by gmariani** [godaddy-wordpress/coblocks#2348]
### Describe the bug:
https://wordpress.org/support/topic/problem-carousel-block-after-update-2/
Issue has been open for over a month with no traction so I decided to open my own ticket on this. I originally opened this ticket regarding 2.20, and am now on 2.22.9

The issue as it stands now is that when going to https://linglestownlife.org/nursery-school/ the carousel doesn't wrap by default like it used to. You can drag and slide it to one side and eventually the images stop. You can even drag it until you see no images, nothing stops it. The navigation buttons are also hidden behind the images.

Up until version 2.20 this worked as expected, after 2.20, all the settings changed and now behaves completely differently and incorrectly. I did eventually find there is a Loop setting which lets it wrap like it used to, but that seems to have changed when upgrading to 2.20. There are also many other users complaining about the carousel functionality on the support topic page linked above.

### To reproduce:
Drop the Carousel block on a page, add 4 images. 

= Carousel settings =
Set the Size to L, Desktop Gutter to 5, Mobile Gutter to 15. Rounded Corners to 0, height in pixels to 250. Disable Lightbox, disable thumbnails.

= Slider settings =
Loop: yes
Autoplay: no
Draggable: yes
Free scroll: no
Arrow navigation: yes
Dot navigation: no
Align cells: no

### Expected behavior:
To be able to drag slides and stop once it reaches the end of the images (when loop is disabled). To have navigation buttons placed above the images. Basically behave like a normal carousel...

### Screenshots:
https://linglestownlife.org/nursery-school/

### Isolating the problem:

- [ ?] This bug happens with no other plugins activated
- [ ?] This bug happens with a default WordPress theme active
- [x ] This bug happens **without** the Gutenberg plugin active
- [x ] I can reproduce this bug consistently using the steps above

### WordPress version:
5.9.2

### Gutenberg version:
n/a

**Comment by rhizome** [comment]
CoBlocks v2.22.10 still broken. However, like in the thread, my issue is that [the carousel is grayed out](https://ibb.co/v4Vg3mV) and the non-grayed center image does not appear. It will loop through them in the form seen in the screenshot, one at a time in a changing pair arrangement. I'll have to switch to single static images until this is fixed unless I can find a way to downrev.

**Comment by gmariani** [comment]
CoBlocks v2.23.1 still broken. Using WordPress 5.9.3, and the buttons are still hidden behind the images. Does anybody test the blocks they work on?

All it needs is this:

.nav-button__next, .nav-button__prev {
z-index:2;
}

because you have 

.swiper-wrapper {
z-index:1;
}

so it sits on top of the buttons but the buttons have no CSS to tell them to be above the wrapper...

**Comment by adamf59** [comment]
Fixing in https://github.com/godaddy-wordpress/coblocks/tree/Issue-2384-carousel-scroll-behavior

**Comment by Zarz29** [comment]
Hi!  So I'm having the same problem as [rhizome](https://github.com/rhizome), and I'm not seeing any mention of it anywhere else, so I thought I'd jump in here.  I have a carousel, and regardless of anything I've tried to fix it, the images are grayed out.  Creating new carousels, they default to being fully grayed out as well.  I just updated to CoBlocks version 2.24.2, and that didn't help.  Any suggestions on getting this resolved? (Or some alternate way I'm supposed to report this? I'm completely new here!)
